src/Console/InstallCommand.php

```diff
@@ -77,9 +94,7 @@ protected function gatherServicesWithSymfonyMenu()
     protected function buildDockerCompose(array $services)
     {
         $depends = collect($services)
-            ->filter(function ($service) {
-                return in_array($service, ['mysql', 'pgsql', 'mariadb', 'redis', 'meilisearch', 'minio', 'selenium']);
-            })->map(function ($service) {
+            ->map(function ($service) {
```

Comment:
This was missing the `memcached` service, but now that we're validating the services, I don't think we need to filter it.